mne/preprocessing/tests/test_ica.py

```diff
@@ -130,6 +130,33 @@ def test_ica_rank_reduction():
 
 
 @requires_sklearn
+def test_ica_reset():
+    """Test ICA resetting"""
+    raw = io.Raw(raw_fname, preload=True).crop(0, stop, False).crop(0.5)
```

Comment:
Done -- PR opened into your branch @dengemann. The changes even speed up tests by ~20% by not preloading extra data and using `.preload_data()` after cropping (hooray!).